python/cudf/cudf/utils/cudautils.py

```diff
@@ -608,6 +608,26 @@ def apply_round(data, decimal):
     return output_dary
 
 
+@cuda.jit
+def gpu_clip(in_col, out_col, lower, upper):
+    i = cuda.grid(1)
+
+    if i < in_col.size:
+        if in_col[i] < lower:
+            out_col[i] = lower
+        elif in_col[i] > upper:
+            out_col[i] = upper
+        else:
+            out_col[i] = in_col[i]
+
+
+def apply_clip(data, lower, upper, inplace):
+    output_dary = data if inplace else rmm.device_array_like(data)
+    if data.size > 0:
+        gpu_clip.forall(data.size)(data, output_dary, lower, upper)
+    return output_dary
```

Comment:
This should only return if `inplace` is `False`. If `inplace` is `True` this function shouldn't return anything.

Reply:
Got it!

Reply:
Will do tomorrow